retro docs review (#4305)
camunda/camunda-docs: docs/self-managed/concepts/multi-region/dual-region.md

  - **26500** for communication to the Zeebe Gateway from clients/workers.
  - **26501** and **26502** for communication between Zeebe brokers and Zeebe Gateway.

### Zeebe cluster configuration

> Only a combination for Zeebe broker counts and replication factors are supported:

Reviewer: This reads to me as  "Only a combination ~~for Zeebe broker counts and replication factors~~ **is** supported". 

Not sure if it would read better with different phrasing: 

```suggestion
Only the following combinations of Zeebe broker counts and replication factors are supported:
```
